spec/react-masonry-component-test.js

```diff
@@ -326,12 +326,16 @@ describe('React Masonry Component', function() {
       }
 
       wrapperContext.setState({items: childrenElements.slice().reverse()});
-      const secondElements = div.querySelectorAll('.item');
 
-      for (let i = 0; i < secondElements.length; i++) {
-        expect(secondElements[i].style.left).toEqual(secondPositions[i].left + 'px');
-        expect(secondElements[i].style.top).toEqual(secondPositions[i].top + 'px');
-      }
+      setTimeout(() => {
+        const secondElements = div.querySelectorAll('.item');
+
+        for (let i = 0; i < secondElements.length; i++) {
+          expect(secondElements[i].style.left).toEqual(secondPositions[i].left + 'px');
+          expect(secondElements[i].style.top).toEqual(secondPositions[i].top + 'px');
+        }
+        // done();
```

Comment:
@eiriklv - I updated this test to account for masonry's default transitionDuration. I will look into it further, but there seems to be a timing issue when the done callback is utilized here (test fails).